src/components/Application.tsx

```diff
@@ -157,8 +157,14 @@ export class Application extends React.Component<Props, State> {
     if (this.state.isLoggedIn && !this.state.isSessionExpired) {
       this.initializeServices()
       this.startBackgroundTasks()
-      this.refreshRecords()
-          .then(this.importJobsIfNeeded.bind(this))
+      this.refreshRecords().then(() => {
+        // Load selected feature if it isn't already (e.g., page refresh w/ jobId).
+        let [jobId] = this.state.route.jobIds
+
+        if (jobId && !this.state.selectedFeature) {
+          this.state.selectedFeature = this.state.jobs.records.find(job => job.id === jobId)
+        }
+      }).then(this.importJobsIfNeeded.bind(this))
```

Comment:
This fixes the job refresh bug (a different ticket) so i'm surprised this wasn't merged in through a different PR.  Maybe it was merged in master but forgot to be merged into UC-Stable?

It should be fine to leave it, as long as there weren't any other changes related to that fix